sounddevice.py

```diff
@@ -2265,10 +2265,66 @@ def reset(self):
 if not hasattr(_ffi, 'I_AM_FAKE'):
     # This object shadows the 'default' class, except when building the docs.
     default = default()
-
+    # A simple enumeration of the Portaudio error codes
+    PaErrorCode = type('PaErrorCode', (), _ffi.typeof('PaErrorCode').relements)
 
 class PortAudioError(Exception):
-    """This exception will be raised on PortAudio errors."""
+    """This exception will be raised on PortAudio errors.
+
+    Parameters
+    ----------
+    msg : optional
+        User error message.
+    errcode : int or None, optional
```

Comment:
probably documenting of the constructor arguments is unnecessary. I could remove them. The attribute documentation OTOH I think is useful for users.